css/main-title.css

```diff
@@ -1,27 +1,29 @@
-.container-title {
+.comp-main-title {
   text-align: center;
-  background-color: white;
   padding-bottom: 60px;
-  padding-top: 135px;
 }
 
-.main-title {
-  height: 297px;
+.padding-top-100 {
+  padding-top: 100px;
+}
+
+.padding-bottom-100 {
+  padding-bottom: 100px;
```

Comment:
Phần padding-100 này không phải của main-title. Do đó không bỏ vào file main-title.css
Code này dùng chung cho toàn bộ các sang và các component khác cũng vậy
Di chuyển những đoạn code không liên quan vào file style.css nha e.